central/processlisteningonport/datastore/datastore_impl.go

```diff
@@ -36,6 +37,10 @@ var (
 	log     = logging.LoggerForModule()
 )
 
+const (
+	getBatchSize = 10000
```

Comment:
If you did not, it may be worth tinkering with this value and comparing benchmark runs.